client/src/fabric/FabricRuntime.ts

```diff
@@ -77,6 +82,81 @@ export class FabricRuntime extends EventEmitter {
         return this.state;
     }
 
+    public async create(): Promise<void> {
+
+        // Delete any existing runtime directory, and then recreate it.
+        await fs.remove(this.path);
+        await fs.ensureDir(this.path);
+
+        // Use Yeoman to generate a new network configuration.
+        await YeomanUtil.run('fabric:network', {
+            destination: this.path,
+            name: this.name,
+            dockerName: this.dockerName,
+            orderer: this.ports.orderer,
+            peerRequest: this.ports.peerRequest,
+            peerChaincode: this.ports.peerChaincode,
+            certificateAuthority: this.ports.certificateAuthority,
+            couchDB: this.ports.couchDB,
+            logspout: this.ports.logs
+        });
+
+    }
+
+    public async importWalletsAndIdentities(): Promise<void> {
+
+        // Ensure that all wallets are created and populated with identities.
+        const fabricWalletGenerator: IFabricWalletGenerator = FabricWalletGeneratorFactory.createFabricWalletGenerator();
+        const walletNames: string[] = await this.getWalletNames();
+        for (const walletName of walletNames) {
```

Comment:
why would there ever be more than one of these for a runtime?

Reply:
@heatherlp today no, but planning for the future where you might import the "commercial paper advanced" network sample which would have a wallet for each organisation.